public/admin/index.html

```diff
@@ -0,0 +1,37 @@
+<!--
+  ~  Copyright 2021 Data and Service Center for the Humanities - DaSCH.
+  ~
+  ~  Licensed under the Apache License, Version 2.0 (the "License");
+  ~  you may not use this file except in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~
+  ~  http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  ~
+  -->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset='utf-8'>
+  <meta name='viewport' content='width=device-width,initial-scale=1'>
+
+  <link rel='icon' type='image/png' href='favicon.png'>
+  <link rel='stylesheet' href='global.css'>
+  <link rel='stylesheet' href='/build/bundle.css'>
+
+  <script defer src='/build/bundle.js'></script>
+
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <!-- second script moved to separate file -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-9MD8RKZJHE"></script>
```

Comment:
we don't use google analytics anymore. can you please remove this.

Reply:
removed in a6606a5